src/core/config/README.md

```
@@ -77,16 +77,16 @@ export XLIO_INLINE_CONFIG="resources.memory_limit=4GB"
 
 **Resolution Rules:**
 - Unique leaf names resolve automatically (e.g., `memory_limit` → `core.resources.memory_limit`)
-- Ambiguous names require enough path to disambiguate (e.g., `daemon.enable` instead of just `enable`)
+- Ambiguous names require enough path to disambiguate (e.g., `hugepages.enable` instead of just `enable`)
 - Full paths always work
 
 **Helpful Error Messages:**
 When a short name is ambiguous, XLIO provides helpful suggestions:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)